plugins/woocommerce/includes/wc-template-functions.php

```diff
@@ -1567,7 +1571,15 @@ function woocommerce_catalog_ordering( $attributes = null ) {
 		if ( ! wc_get_loop_prop( 'is_paginated' ) || ! woocommerce_products_will_display() ) {
 			return;
 		}
-		$show_default_orderby    = 'menu_order' === apply_filters( 'woocommerce_default_catalog_orderby', get_option( 'woocommerce_default_catalog_orderby', 'menu_order' ) );
+
+		/**
+		 * Filter the default catalog orderby.
+		 *
+		 * @since 1.6.4
+		 *
+		 * @param string $default_orderby The default orderby option.
+		 */
+		$show_default_orderby = 'menu_order' === apply_filters( 'woocommerce_default_catalog_orderby', get_option( 'woocommerce_default_catalog_orderby', 'menu_order' ) );
```

Comment:
should this change part of this PR?

Reply:
Not related but the linter kept complaining that a comment there was necessary 🤔